content/blog/secure-software-development/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Secure Software Development Framework (SSDF) at Adoptium
+date: "2022-10-07T12:00:00+00:00"
```

Comment:
The date should be for this week, no?

Reply:
This PR was created some time ago - I will update it when it's approved and published/ready to publish, otherwise I'd be updating it every day :-)